region/brinstar/pink/Waterway Energy Tank Room.json

```diff
@@ -97,6 +97,60 @@
     }
   ],
   "strats": [
+    {
+      "link": [1, 1],
+      "name": "R-Mode Spark Interrupt (Gain Blue Suit)",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        {"or": [
+          "h_CrystalFlash",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 5}}
+          ]}
+        ]},
+        {"or":[
+          {"and":[
+            "Gravity",
+            "h_shinechargeMaxRunway"
+          ]},
+          {"and":[
+            {"or":[
+              "SpaceJump",
+              {"and": [
+                "canSuitlessMaridia",
+                {"or":[
+                  "canPreciseWalljump",
+                  "canBombJumpWaterEscape",
+                  {"and": [
+                    "h_useSpringBall",
+                    "canJumpIntoIBJ"
+                  ]},
+                  {"and": [
+                    "canBePatient",
+                    "canTrickyUseFrozenEnemies",
+                    {"ammo": {"type": "Super", "count": 1}}
+                  ]}
+                ]}
+              ]}
+            ]},
+            {"canShineCharge": {"usedTiles": 32, "openEnd": 1}}
+          ]}
+        ]},
+        {"autoReserveTrigger": {"maxReserveEnergy": 95}},
```

Comment:
You probably want to give some dummy real energy, from farming the fish, so you can reload the room incase of heat on the other side.

Reply:
Added 20 energy for 3 Puyos. I don't trust the Skulltera's drop rate enough.